authors/apps/authentication/tests/test_functionality.py

```diff
@@ -0,0 +1,56 @@
+from django.urls import reverse
+from rest_framework.test import APITestCase
+from rest_framework import status
+from authors.apps.authentication.models import User
+
+
+class Authentication(APITestCase):
+
+    def test_for_new_user(self):
+        # url = reverse('signup')
```

Comment:
Be consistent with your comments on functions, don't leave other functions without comments